JuiceMaker/JuiceMaker/Controller/ViewController.swift

```diff
@@ -1,16 +1,122 @@
+        pineappleStockLabel.text = JuiceMaker.sharedJuiceMaker.currentFruitStock(of: .pineapple)
+        kiwiStockLabel.text = JuiceMaker.sharedJuiceMaker.currentFruitStock(of: .kiwi)
+        mangoStockLabel.text = JuiceMaker.sharedJuiceMaker.currentFruitStock(of: .mango)
+    }
+    
+    @IBAction func ddalbaJuiceButtonTapped(_ sender: UIButton) {
+        let orderedJuice: JuiceMenu = .strawberryBananaJuice
+        if JuiceMaker.sharedJuiceMaker.makeJuice(orderedJuice) {
+            updateStockLabel()
+            successAlert(name: orderedJuice.juiceName)
+        } else {
+            failAlert()
+        }
+    }
+    
+    @IBAction func strawberryJuiceButtonTapped(_ sender: UIButton) {
+        let orderedJuice: JuiceMenu = .strawberryJuice
+        if JuiceMaker.sharedJuiceMaker.makeJuice(orderedJuice) {
+            updateStockLabel()
+            successAlert(name: orderedJuice.juiceName)
+        } else {
+            failAlert()
+        }
+    }
+    
+    @IBAction func bananaJuiceButtonTapped(_ sender: UIButton) {
+        let orderedJuice: JuiceMenu = .bananaJuice
+        if JuiceMaker.sharedJuiceMaker.makeJuice(orderedJuice) {
+            updateStockLabel()
+            successAlert(name: orderedJuice.juiceName)
+        } else {
+            failAlert()
+        }
+    }
+    
+    @IBAction func pineappleJuieButtonTapped(_ sender: UIButton) {
+        let orderedJuice: JuiceMenu = .pineappleJuice
+        if JuiceMaker.sharedJuiceMaker.makeJuice(orderedJuice) {
+            updateStockLabel()
+            successAlert(name: orderedJuice.juiceName)
+        } else {
+            failAlert()
+        }
     }
     
+    @IBAction func kiwiJuiceButtonTapped(_ sender: UIButton) {
+        let orderedJuice: JuiceMenu = .kiwiJuice
+        if JuiceMaker.sharedJuiceMaker.makeJuice(orderedJuice) {
+            updateStockLabel()
+            successAlert(name: orderedJuice.juiceName)
+        } else {
+            failAlert()
+        }
+    }
+    
+    @IBAction func mangoJuiceButtonTapped(_ sender: UIButton) {
+        let orderedJuice: JuiceMenu = .mangoJuice
+        if JuiceMaker.sharedJuiceMaker.makeJuice(orderedJuice) {
+            updateStockLabel()
+            successAlert(name: orderedJuice.juiceName)
+        } else {
+            failAlert()
+        }
+    }
+    
+    @IBAction func mankiJuiceButtonTapped(_ sender: UIButton) {
+        let orderedJuice: JuiceMenu = .mangoKiwiJuice
+        if JuiceMaker.sharedJuiceMaker.makeJuice(orderedJuice) {
+            updateStockLabel()
+            successAlert(name: orderedJuice.juiceName)
+        } else {
+            failAlert()
+        }
+    }
+    
+    @IBAction func barButtonTapped(_ sender: UIBarButtonItem) {
+        performSegue(withIdentifier: "toSecondVC", sender: self)
+    }
+    
+    func successAlert(name: String) {
+        let alert = UIAlertController(title: "\(name) 나왔습니다! 맛있게 드세요!",message: nil, preferredStyle: UIAlertController.Style.alert)
```

Comment:
```suggestion
        let alert = UIAlertController(title: "\(name) 나왔습니다! 맛있게 드세요!",message: nil, preferredStyle: .alert)
```

참고로 타입이 명확하기 때문에 위처럼 타입명은 생략하고 사용가능합니다!